lib/Dialect/TensorExt/Transforms/Passes.td

```diff
@@ -157,4 +157,114 @@ def AlignTensorSizes : Pass<"align-tensor-sizes"> {
+    %8 = arith.muli %arg0, %cst_2 : tensor<16xi32>
+    %c8_i32 = arith.constant 8 : i32
+    %9 = tensor_ext.rotate %8, %c8_i32 : tensor<16xi32>, i32
+    %10 = arith.addi %7, %9 : tensor<16xi32>
+    %cst_3 = arith.constant dense<[0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0]> : tensor<16xi32>
+    %11 = arith.muli %arg0, %cst_3 : tensor<16xi32>
+    %c1_i32_4 = arith.constant 1 : i32
+    %12 = tensor_ext.rotate %11, %c1_i32_4 : tensor<16xi32>, i32
+    %cst_5 = arith.constant dense<[0, 0, 1, 0, 0, 1, 0, 1, 0, 1, 1, 0, 1, 0, 0, 0]> : tensor<16xi32>
+    %13 = arith.muli %arg0, %cst_5 : tensor<16xi32>
+    %c2_i32_6 = arith.constant 2 : i32
+    %14 = tensor_ext.rotate %13, %c2_i32_6 : tensor<16xi32>, i32
+    %15 = arith.addi %12, %14 : tensor<16xi32>
+    %cst_7 = arith.constant dense<[0, 0, 1, 0, 0, 1, 0, 0, 0, 1, 1, 0, 0, 0, 1, 0]> : tensor<16xi32>
+    %16 = arith.muli %arg0, %cst_7 : tensor<16xi32>
+    %c4_i32_8 = arith.constant 4 : i32
+    %17 = tensor_ext.rotate %16, %c4_i32_8 : tensor<16xi32>, i32
+    %18 = arith.addi %15, %17 : tensor<16xi32>
+    %cst_9 = arith.constant dense<[0, 0, 1, 0, 0, 1, 1, 0, 0, 0, 1, 0, 0, 0, 0, 0]> : tensor<16xi32>
+    %19 = arith.muli %arg0, %cst_9 : tensor<16xi32>
+    %c8_i32_10 = arith.constant 8 : i32
+    %20 = tensor_ext.rotate %19, %c8_i32_10 : tensor<16xi32>, i32
+    %21 = arith.addi %18, %20 : tensor<16xi32>
+    %22 = arith.addi %10, %21 : tensor<16xi32>
+    %cst_11 = arith.constant dense<[0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 0]> : tensor<16xi32>
+    %23 = arith.muli %arg0, %cst_11 : tensor<16xi32>
+    %c1_i32_12 = arith.constant 1 : i32
+    %24 = tensor_ext.rotate %23, %c1_i32_12 : tensor<16xi32>, i32
+    %cst_13 = arith.constant dense<[0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 1]> : tensor<16xi32>
+    %25 = arith.muli %arg0, %cst_13 : tensor<16xi32>
+    %c2_i32_14 = arith.constant 2 : i32
+    %26 = tensor_ext.rotate %25, %c2_i32_14 : tensor<16xi32>, i32
+    %27 = arith.addi %24, %26 : tensor<16xi32>
+    %cst_15 = arith.constant dense<[0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1]> : tensor<16xi32>
+    %28 = arith.muli %arg0, %cst_15 : tensor<16xi32>
+    %c4_i32_16 = arith.constant 4 : i32
+    %29 = tensor_ext.rotate %28, %c4_i32_16 : tensor<16xi32>, i32
+    %30 = arith.addi %27, %29 : tensor<16xi32>
+    %cst_17 = arith.constant dense<[0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0, 1]> : tensor<16xi32>
+    %31 = arith.muli %arg0, %cst_17 : tensor<16xi32>
+    %c8_i32_18 = arith.constant 8 : i32
+    %32 = tensor_ext.rotate %31, %c8_i32_18 : tensor<16xi32>, i32
+    %33 = arith.addi %30, %32 : tensor<16xi32>
+    %34 = arith.addi %22, %33 : tensor<16xi32>
+    return %34 : tensor<16xi32>
+  }
+  ```
+  }];
+
+  let dependentDialects = ["mlir::heir::tensor_ext::TensorExtDialect"];
+
+  let options = [
+    Option<
+      "ciphertextSize",
+      "ciphertext-size",
```

Comment:
ugh our naming here is diverging. could you file a placeholder issue to resolve the names ciphertext-degree, tiling-size, and ciphertext-size?

Reply:
Done